[
    {
        "problem_type": "Test Failure",
        "test_name": "TestQueueFreeze.test_create_queue_freeze_without_cascading_effect",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/api/test_queue_freeze.py",
        "lineno": null,
        "error": "AssertionError",
        "test_framework": "pytest",
        "stack_trace": "AssertionError: assert 'The pull request is the 2nd in the queue to be merged' == 'The pull request is the 1st in the queue to be merged'"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestQueueCommand.test_queue_command_unqueue_pr_after_ci_failure",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/commands/test_queue.py",
        "lineno": null,
        "error": "AssertionError",
        "test_framework": "pytest",
        "stack_trace": "assert 'The pull request has been removed from the queue' in '...'"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_downloading",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "MissingEventTimeout",
        "test_framework": "pytest",
        "stack_trace": "MissingEventTimeout: Never got event `workflow_job` with payload `{'action': 'completed'}` (timeout)"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_downloading_with_matrix",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "MissingEventTimeout",
        "test_framework": "pytest",
        "stack_trace": "MissingEventTimeout: Never got event `workflow_job` with payload `{'action': 'completed'}` (timeout)"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_downloading_with_no_steps",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "AssertionError",
        "test_framework": "pytest",
        "stack_trace": "assert None is not None"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_embedding",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "MissingEventTimeout",
        "test_framework": "pytest",
        "stack_trace": "MissingEventTimeout: Never got event `workflow_job` with payload `{'action': 'completed'}` (timeout)"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_embedding_matrix",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "MissingEventTimeout",
        "test_framework": "pytest",
        "stack_trace": "MissingEventTimeout: Never got event `workflow_job` with payload `{'action': 'completed'}` (timeout)"
    },
    {
        "problem_type": "Test Failure",
        "test_name": "TestLogEmbedderGithubAction.test_log_step_name_to_zip_filenames",
        "language": "Python",
        "filename": "mergify_engine/tests/functional/log_embedder/test_github_action.py",
        "lineno": null,
        "error": "MissingEventTimeout",
        "test_framework": "pytest",
        "stack_trace": "MissingEventTimeout: Never got event `workflow_job` with payload `{'action': 'completed'}` (timeout)"
    }
]